thoth/storages/graph/dgraph.py

```diff
@@ -704,6 +705,7 @@ def get_all_versions_python_package(
             PythonPackageVersion.get_label(),
             package_name,
             q,
+            "@cascade" if not only_known_index else ""
```

Comment:
the default case should be the one without @cascade because the function might be implemented already in other places ```"@cascade" if only_known_index else ""```

Reply:
Ah good eye, thanks! I had there the `allow_missing_values` before and refactored incorrectly.

Reply:
No problem :) well done @CermakM :+1: